src/commands.cpp

```diff
@@ -60,13 +52,12 @@ CommandProgress BaseCommand::m_commandProgress;
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 BaseCommand::BaseCommand(BaseCommand::CrossViewType crossViewType, SketchWidget* sketchWidget, QUndoCommand *parent)
-	: QUndoCommand(parent)
+	: QUndoCommand(parent),
+    m_crossViewType(crossViewType),
+    m_sketchWidget(sketchWidget),
+    m_parentCommand(parent),
+    m_index(BaseCommand::nextIndex++)
```

Comment:
Here (and some other places) please use tabs for indentation.

Reply:
I'll fix that, my vimrc is responsible for the mismatch, looks correct on my side but it actually isn't.